firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -271,6 +275,14 @@ String getName() {
     return Tasks.call(backgroundExecutor, this::deleteFirebaseInstallationId);
   }
 
+  /** Register a callback {@link FidListener} to receive fid changes. */
+  @Override
+  public void registerFidListener(@Nullable FidListener listener) {
```

Comment:
Should it instead be?
```suggestion
  public void registerFidListener(@NonNull FidListener listener) {
```